crates/orchestrator/src/tests/server/job_routes.rs

```diff
@@ -0,0 +1,102 @@
+use std::io::Read;
+use std::net::SocketAddr;
+use std::sync::Arc;
+
+use axum::http::StatusCode;
+use chrono::{SubsecRound as _, Utc};
+use hyper::body::Buf;
+use hyper::{Body, Request};
+use rstest::*;
+use starknet::providers::jsonrpc::HttpTransport;
+use starknet::providers::JsonRpcClient;
+use url::Url;
+use utils::env_utils::get_env_var_or_default;
+use uuid::Uuid;
+
+use crate::config::{init_config, Config};
+use crate::jobs::types::{ExternalId, JobItem, JobStatus, JobType};
+use crate::queue::init_consumers;
+use crate::routes::job_routes::job_routes;
+use crate::tests::config::TestConfigBuilder;
+
+#[fixture]
+pub async fn setup_job_server() -> (SocketAddr, Arc<Config>) {
+    let provider = JsonRpcClient::new(HttpTransport::new(
+        Url::parse("http://localhost:9944".to_string().as_str()).expect("Failed to parse URL"),
```

Comment:
let's move this url to env?

Reply:
Valid, changed [here](https://github.com/madara-alliance/madara-orchestrator/pull/164/commits/ec1f2470778c1e75b478f2411eba08bfba300075).